web/template/attorney_enter_reference_number.gohtml

```diff
@@ -10,9 +10,12 @@
       {{ trHtml .App "attorneyReferenceNumberContent" }}
 
       <form novalidate method="post">
-        {{ template "input" (input . "reference-number" "attorneyReferenceNumber" .Form.ReferenceNumberRaw "classes" "govuk-input--width-20 govuk-!-margin-bottom-3 govuk-input--extra-letter-spacing" "hint" "referenceNumberHint")}}
-        
-        {{ template "continue-button" . }}
+        <div class="govuk-form-group">
+          <fieldset class="govuk-fieldset">
```

Comment:
we don't need a form-group or fieldset here as the `input` template handles it

Reply:
My bad ✅